_posts/2020-06-25-huechong-hettrick.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Software is the backbone"
+author: "@pweschmidt"
+rse: "Neil Chue Hong and Simon Hettrick"
+phenotype: neilsimon-rse-phenotype.png
+excerpt: "For ca 10 years the Software Sustainability Institute in the UK has been the backbone of research enginnering. But how did they fare during the coronavirus pandemic and how will research software engineering change in future?"
```

Comment:
Did you mean to say “ca” ?

Reply:
Looks like engineering is spelled wrong.

Reply:
“In the future”

Reply:
Thanks for spotting the typo in engineering. "Ca" is short for circa, will replace it with about. "In Future" actually works I think.